src/main/java/org/broadinstitute/consent/http/models/DatasetUpdate.java

```diff
@@ -1,12 +1,24 @@
 package org.broadinstitute.consent.http.models;
 
+import com.google.gson.Gson;
+import com.google.gson.ToNumberPolicy;
 import java.util.List;
+import org.broadinstitute.consent.http.util.gson.GsonUtil;
 
-public record DatasetUpdate (
+public record DatasetUpdate(
     String name,
     Integer dacId,
     List<DatasetProperty> properties
-){
+) {
+
+  private static final Gson GSON = GsonUtil.gsonBuilderWithAdapters().setObjectToNumberStrategy(
+      ToNumberPolicy.LONG_OR_DOUBLE).create();
```

Comment:
Should we move `setObjectToNumberStrategy` into `gsonBuilderWithAdapters` globally? That way we do not need to remember to depend on this particular `GSON` object for any Dataset related methods.

Reply:
That's a great point, thank you!